kong/core/globalpatches.lua

```diff
@@ -124,9 +124,13 @@ return function(options)
         end
         return true
       end
-      function SharedDict:incr(key, value)
+      function SharedDict:incr(key, value, init)
         if not self.data[key] then
-          return nil, "not found"
+          if not init then
+            return nil, "not found"
+          else
+            self.data[key].value = init
+          end
```

Comment:
The above change is not correct. It will set the initial value and immediately apply an increment (in the code below), whereas is should return the value of `init` without applying an increment.

correct?